internal/app/init/internal/reg/reg.go

```diff
@@ -72,7 +73,19 @@ func (r *Registrator) Upgrade(ctx context.Context, in *proto.UpgradeRequest) (da
 		return data, err
 	}
 
+	// stop kubelet
```

Comment:
should we also stop kubeadm as well? (it shouldn't be running, but to be 100% sure)